src/item.cpp

```diff
@@ -2646,10 +2664,56 @@ std::string item::info( std::vector<iteminfo> &info, const iteminfo_query *parts
         }
 
         // TODO: Unhide when enforcing limits
-        if( is_bionic() && get_option < bool >( "CBM_SLOTS_ENABLED" )
-            && parts->test( iteminfo_parts::DESCRIPTION_CBM_SLOTS ) ) {
-            info.push_back( iteminfo( "DESCRIPTION", list_occupied_bps( type->bionic->id,
-                                      _( "This bionic is installed in the following body part(s):" ) ) ) );
+        if( is_bionic() ) {
+            if( get_option < bool >( "CBM_SLOTS_ENABLED" )
+                && parts->test( iteminfo_parts::DESCRIPTION_CBM_SLOTS ) ) {
+                info.push_back( iteminfo( "DESCRIPTION", list_occupied_bps( type->bionic->id,
+                                          _( "This bionic is installed in the following body part(s):" ) ) ) );
+            }
+            insert_separation_line();
+            const bionic_id bid( typeId() );
+
+            if( !bid->encumbrance.empty() ) {
+                std::ostringstream encumb;
+                for( const auto &element : bid->encumbrance ) {
+                    encumb << body_part_name_as_heading( element.first,
+                                                         1 ) << ": " << "<color_yellow>" << element.second << "</color>" << " ";
+                }
+                info.push_back( iteminfo( "DESCRIPTION", _( "<bold>Encumbrance:</bold> " ) + encumb.str() ) );
+            }
+
+            if( !bid->env_protec.empty() ) {
+                std::ostringstream Ep;
+                for( const auto &element : bid->env_protec ) {
+                    Ep << body_part_name_as_heading( element.first,
+                                                     1 ) << ": " << "<color_yellow>" << element.second << "</color>" << " ";
+                }
+                info.push_back( iteminfo( "DESCRIPTION",
+                                          _( "<bold>Environmental Protection:</bold> " ) + Ep.str() ) );
+            }
+
+            const float weight_bonus = bid->weight_capacity_bonus;
+            const float weight_modif = bid->weight_capacity_modifier;
+            if( weight_modif != 1 ) {
+                std::string modifier;
+                if( weight_modif < 1 ) {
+                    modifier = string_format( "<bad>x%.2f</bad>", weight_modif );
+                } else {
+                    modifier = string_format( "<good>x%.2f</good>", weight_modif );
+                }
+                info.push_back( iteminfo( "DESCRIPTION",
+                                          _( "<bold>Weight capacity modifier</bold>: " ) + modifier ) );
```

Comment:
This should follow the form of other numeric values like "weight" itself, e.g. 
`iteminfo( "...", "weight modifier", "<num>", iteminfo::no_newline, modifier )`
This format allows the "compare item" dialog to colorize the numeric values according to their relation (the better value is colored green and the worse is red). But this does not work if you just give a string, not a numeric value.

Reply:
I've tried :
```cpp
 info.push_back( iteminfo( "DESCRIPTION", body_part_name_as_heading( element.first, 1 ), "", iteminfo::no_flags, element.second ) );
```
But it doesn't display the value... It's `iteminfo( "category", "first string", "string before the value", flag, value)` right? Do I have to use double and not int as value?

Reply:
Here's good example:
https://github.com/CleverRaven/Cataclysm-DDA/blob/59d443ef1841657870c80262910f533f73def6c8/src/item.cpp#L1080-L1086

Reply:
>  Do I have to use double and not int as value?

Doesn't matter, will be converted automatically to `double` as that is what the function requires. 

> I've tried :
> But it doesn't display the value...

You missed the "<num>" part. The "<num>" string is replaced with the numeric value that you provided see the example from AMurkin.